packages/manager/modules/storages/src/netapp/netapp.template.html

```diff
@@ -0,0 +1,123 @@
+<div class="" style="overflow: auto;">
+    <oui-header
+        data-heading="{{:: 'netapp_product_name' | translate }} - {{ $ctrl.netappId }}"
+    >
+        <oui-header-tabs>
+            <oui-header-tabs-item
+                data-href="{{ $ctrl.serviceLink }}"
+                data-active="$ctrl.currentActiveLink() === $ctrl.serviceLink"
+                ><span data-translate="netapp_tab_global_news"></span>
+            </oui-header-tabs-item>
+            <oui-header-tabs-item
+                data-href="{{ $ctrl.shareLink }}"
+                data-active="$ctrl.currentActiveLink() === $ctrl.shareLink"
+                ><span data-translate="netapp_tab_share"></span>
+            </oui-header-tabs-item>
+        </oui-header-tabs>
+        <oui-guide-menu data-text="{{:: 'netapp_guides_header' | translate }}">
+            <oui-guide-menu-item href="{{ $ctrl.guideUrl }}" data-external>
+                <span data-translate="netapp_guides_header_all_guides"></span>
+            </oui-guide-menu-item>
+        </oui-guide-menu>
+    </oui-header>
+
+    <div class="px-3 px-md-4" data-ui-view>
+        <div class="row">
+            <div class="col-md-4">
+                <oui-tile
+                    data-heading="{{:: 'service_netapp_global_news' | translate }}"
+                >
+                    <oui-tile-definition
+                        data-term="{{:: 'service_netapp_name' | translate }}"
+                    >
+                        <oui-tile-description>
+                            <span
+                                data-ng-bind="$ctrl.service.displayName"
+                            ></span>
+                        </oui-tile-description>
+                        <oui-action-menu
+                            aria-label="{{:: 'menu_action_label' | translate:{ t0: ('service_netapp_name_modifiy' | translate) } }}"
+                            data-compact
+                            data-placement="end"
+                        >
+                            <oui-action-menu-item
+                                data-ui-sref="app.dedicated.server.dashboard.edit-display-name"
```

Comment:
If possible could this be injected through the state's resolve ? 
An example can be found https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/instances.routing.js#L118